ziti/cmd/ziti/cmd/edge_controller/common.go

```diff
@@ -48,7 +49,9 @@ func mapNameToID(entityType string, val string) (string, error) {
 
 	entity := list[0]
 	entityId, _ := entity.Path("id").Data().(string)
-	fmt.Printf("Found %v with id %v for name %v\n", entityType, entityId, val)
+	if val, found := os.LookupEnv("DEBUG"); found && strings.EqualFold("true", val) {
```

Comment:
Should we standardize these early w/ a prefix? `ZITI_DEBUG`

Reply:
I like it. Renamed to ZITI_CLI_DEBUG